.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Prepare Buildpack Releases
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        description: "Bump"
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  prepare-release:
+    uses: heroku/languages-github-actions/.github/workflows/_buildpacks-prepare-release.yml@main
+    with:
+      bump: ${{ inputs.bump }}
+      declarations_starting_version: 1.0.0
```

Comment:
you could either:
- run the prepare release workflow and review the PR it creates.  this can be closed if you see an issue or are just testing
- download the shared actions repository, build the `actions` binary locally, then run the `prepare-release` command manually in the directory of your repository